src/Statistics.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  color: #383854;
```

Comment:
Why do you need the color white?
it doesn't do anything at the moment

Reply:
This isn't white

Reply:
@SamiSha99 my point is, why you are using `color` when there is no text?